### ISSUE TITLE
Persisting and launching jobs at given intervals

### ISSUE THREAD

**Issue by d-ivashchuk** [agenda/agenda#811]
Hello! First of all thanks to the creator and all maintainers, library seems awesome. I wanted to get a little advice if it's useful in my case because I can't seem to make it work as I expect.

I have a bot which needs to remind a user of the message sent.
```
agenda.on("ready", () => {
  agenda.schedule("in 30 seconds", "remind");
  agenda.start();
});

bot.hears("t", ctx => {
  agenda.define("remind", function(job) {
    ctx.reply("qwe");
  });
});
```

There are couple of questions though, with this code I am only able to wirte one job into mongo, which is not smth I would expect. Technically I want users to be able to sent as many 't' messages which would then generate the jobs to reply to them in 30 seconds. Is it possible with agenda?

Another question is when I set the job with this code, and stop & relaunch server, the job won't be executed. Is it appropriate behaviour? After I send message, the job will be executed twice at once.

I would highly appreciate the advice on these points and if agenda is not the good choice for such use case directions on what to look at, as I am quite new to node ecosystem.

Thanks!

**Comment by d-ivashchuk** [comment]
Got everything working for my case. In case someone has the same use case and wants to know the solution, feel free to dm.

TLDR: Read carefully the api for async/sync jobs, in my case I needed to place job.done() in my code for everything to work properly!
